client/unit-tests/symbolic/collections/test_static_collection.py

```diff
@@ -1,6 +1,6 @@
 import pytest
```

Comment:
can you add more integration tests on the client side showing how these new filter features are intended to be used?